app/mailers/admin_mailer.rb

```diff
@@ -12,4 +12,19 @@ def slot_availability(availability)
       subject: "Slot availability - #{Time.zone.today}"
     )
   end
+
+  def confirmed_bookings(email_address)
+    exporter = WeeklyMetricsConfirmedCsvExporter.new(weeks: 12)
+
+    attachments['confirmed_bookings.csv'] = {
```

Comment:
Should we be checking the size of the attachments? email provider will be rejecting the email if this goes over a certain size, although we should be ok don't think this might go over 25MG.

Reply:
Yes, thought of that. I think we have a long while until we get a large csv